### PR TITLE
Scratch/cloudscan worker and pool endpoints

### PR DESCRIPTION
**Pull request by xolian**
add worker assignment, test, and documentation for cancel job to legacy.

### CODE REVIEW

File: fortifyapi/client.py [hunk 1]
```diff
@@ -389,9 +401,17 @@ def get(self, job_token):
...
+        Manage ScanCentral SAST jobs with state change to CANCEL.  Typical usage
+        would be of a large backlog of PENDING jobs, because no sensor was available or is in a bad state.
+        :param job_token: Scan Central Job Token assigned to the job
+        TODO: fix with return api.post(f"/api/v1/cloudjobs/{job_token}/acion", type="cancel")['data']['status'].  See
+        POST /ssc/api/v1/cloudjobs/1076964f-ec72-48e1-a897-6da9683a75df/action HTTP/1.1
+        {"type":"cancel"}
+        """
         with self._api as api:
-            return api.post(f"/api/v1/cloudjobs/action/cancel", jobTokens=[self['jobToken']])
+            return api.post (f"/api/v1/cloudjobs/action/cancel", jobTokens=[self['jobToken']])
```
Comment (vetsin) [inline fortifyapi/client.py:414]: space is accidental?

File: tests/test_pool.py [hunk 2]
```diff
@@ -64,5 +64,21 @@ def test_list_jobs(self):
...
 
+    def test_assign_worker(self):
+        pool_name = 'unit_test_pool_assign_worker_zz'
+        client = FortifySSCClient(self.c.url, self.c.token)
+        self.c.setup_proxy(client)
+        new_pool = client.pools.create(pool_name)
+        pprint(new_pool)
+
+        # Assign a worker that is unassigned to a pool
+        unassigned_worker = [worker['uuid'] for worker in client.workers.list() if worker['cloudPool'] is None]
```
Comment (vetsin) [inline tests/test_pool.py:75]: this test is predicated on the idea there's an unassigned worker? ergo it's not repeatable. at the least we have to unassign the worker at the start and end of the test so we maintain a constant state

Comment (xolian) [reply tests/test_pool.py:75]: great suggestion.  I'll add an assignment to pool 00000000-0000-0000-0000-000000000001 for this test.

Comment (xolian) [reply tests/test_pool.py]: added test_unassign_worker

File: fortifyapi/client.py [hunk 3]
```diff
@@ -347,15 +348,21 @@ def delete(self):
...
+        :param pool_uuid:
+        :param worker_uuid:
+        :return: ['status'] == success
+        """
         with self._api as api:
-            r = api.post(f"/api/v1/cloudpools/{self['uuid']}/workers/action/assign", {
-                "workerUuids": worker_uuids
+            r = api.post(f"/api/v1/cloudpools/{pool_uuid}/workers/action", {
+                "type": "assign",
+                "ids": [worker_uuid]
```
[the remark this answers is not in the document]
Comment (xolian) [reply fortifyapi/client.py]: added `[worker_uuid] if type(worker_uuid) is str else list(worker_uuid)`

**Pull request merged**